### DISCUSSION TITLE
Jobs stuck in queue state for hours

### DISCUSSION THREAD

**Discussion by anthonator** [community/community#45294] [Actions]
### Select Topic Area

Bug

### Body

We have several jobs that appear to be stuck in the queue state for several hours. Is there a way to get them unstuck?

**Comment by jge162** [comment]
you can try restarting the job by clicking the "Re-run" button on the job page.

If there are a lot of jobs in the queue, clearing it may help the stuck job move forward. You can do this by clicking the "Clear queue" button on the Actions page.

other than that, thats all I know. best
